src/custom-element.js

```diff
@@ -1869,11 +1869,12 @@ function createBaseCustomElementClass(win, elementConnectedCallback) {
 
     /**
      * Get the purpose consents that should be granted.
-     * @return {?Array<string>}
+     * @return {Array<string>=}
```

Comment:
I've only ever seen the optional/default-value syntax (`{Type=}`) used for `@param` annotations, never for return values. I can't find any documentation suggesting it's supported. Even if it is, the nullable syntax `?Type` or explicit union `null|Type` might be clearer

Reply:
Sg!